run-tests.php

```diff
@@ -3167,14 +3169,14 @@ function get_summary(bool $show_ext_summary): string
     return $summary;
 }
 
-function show_start(int $start_time): void
+function show_start(int $start_timestamp): void
 {
-    echo "TIME START " . date('Y-m-d H:i:s', $start_time / 1e9) . "\n=====================================================================\n";
+    echo "TIME START " . date('Y-m-d H:i:s', $start_timestamp) . "\n=====================================================================\n";
 }
 
-function show_end(int $end_time): void
+function show_end(int $start_timestamp, float $start_time, float $end_time): void
```

Comment:
```suggestion
function show_end(int $start_timestamp, int|float $start_time, int|float $end_time): void
```
If using the `as_number` param for `hrtime()` but currently the values are arrays. Which might actually better? As those are guarenteed to be integers even on 32 bit.

Reply:
I changed to int|float. It's easier to read when we get direct nanoseconds to compute.